### DISCUSSION TITLE
Launching Kalidoface - A web based Virtual Youtuber app.

### DISCUSSION THREAD

**Discussion by yeemachine** [guansss/pixi-live2d-display#18] [Show and tell]
Hi!

Wanted to drop in and let you know I've used pixi-live2d-display in my latest webapp, Kalidoface. It animates Live2D models with real time face capture provided by Tensorflow's pretrained models. Users can change their own Live2D model by just dragging and dropping a zip file of the runtime folder. And finally, the app has peer to peer calling to chat using Live2D avatars instead of a video call. 

It's now live at [kalidoface.com](https://kalidoface.com). 

Thanks for being so responsive with all the feature suggestions and bug reports from me over these couple of months and I'll definitely keep following the progress on this library. I've added your repo as credits to my site, but let me know if you'd like me to remove it.

**Comment by guansss** [comment]
Hi. That appears to be an amazing app and I'm glad to get this project to help with it! Also happy to be in the credits. :D

**Reply by yeemachine** [reply]
😄 

By the way, are you aware of any Live2D rigger communities or even Vtuber communities I could ask questions in? I'd love to add some nice features like accessories/predetermined movements that I see other vtubers toggle on and off. Don't really have a model I could test that with though. 

Also, have you gotten your plugin officially recognized yet? I wonder if I could go that route as well.

**Reply by guansss** [reply]
Hmm, I don't really know about these communities as I'm developing this individually, but I've just found [this discord](https://discord.com/invite/4S3Sesj) when searching around, not sure if that fits your need. As for the predetermined movements, I guess they're just normal motions or expressions with some keybindings. You can probably check this [introduction of VTube Studio](https://www.youtube.com/watch?v=Xo774VpWASE) to find the ideas, it seems to be a popular app among vtubers.

About this plugin, on the Pixi side, it's been approved and listed in their wiki quite a while ago, just as an inconspicuous link though. On the Live2D side... they don't seem to approve anything.
